nix/status-go/mobile/build.nix

```diff
@@ -5,7 +5,7 @@
 , platform ? "android"
 , architectures ? [ "arm64" "arm" "x86" ]
 , goBuildFlags ? [ ]
-, goBuildLdFlags ? [ ]
+, goBuildLdFlags ? [ "-X node.OpenseaKeyFromEnv=$OPENSEA_API_KEY" ]
```

Comment:
For one, these defaults are overridden in `nix/status-go/default.nix`, so this does nothing.

Secondly, the secret is not even available in Nix because of sandboxing. See how we handle other env secrets:
https://github.com/status-im/status-react/blob/7f007f204f33536c41a9f0fb3e0501ae20cbc337/scripts/release-android.sh#L52-L54
https://github.com/status-im/status-react/blob/7f007f204f33536c41a9f0fb3e0501ae20cbc337/nix/mobile/android/jsbundle/default.nix#L47-L53

Reply:
I made a mistake in https://github.com/status-im/status-jenkins-lib/commit/81e8d8bf because I didn't add `OPENSEA_API_KEY` to [`keepEnv`](https://github.com/status-im/status-jenkins-lib/blob/81e8d8bfc6ce9b00ea20401117b2da38f2618303/vars/android.groovy#L62-L68).

So we'll have to fix that too.

Reply:
Here's the change: https://github.com/status-im/status-jenkins-lib/pull/39

Reply:
@jakubgs approved that PR

So now I have to move this line to `nix/status-go/default.nix`, correct?

Reply:
funny thing is that @churik tested it and said it works :D it likely worked for some other reason

Reply:
No, that should not work. Nor does it work:
```diff
diff --git a/nix/status-go/mobile/build.nix b/nix/status-go/mobile/build.nix
index e77a6e3de..cc8f5cad2 100644
--- a/nix/status-go/mobile/build.nix
+++ b/nix/status-go/mobile/build.nix
@@ -62,6 +62,7 @@ in buildGo116Package {
 
     echo -e "\nBuilding for targets: ${concatStringsSep "," targetArchs}\n"
 
+    echo "OPENSEA_API_KEY: $OPENSEA_API_KEY"
     ${gomobile}/bin/gomobile bind \
       -target=${concatStringsSep "," targetArchs} \
       -ldflags="${CGO_LDFLAGS}" \
```
```log
 > export OPENSEA_API_KEY=TEST_VALUE

 > make status-go-android           
nix/scripts/build.sh targets.status-go.mobile.android
Running: nix-build --pure --fallback --no-out-link --show-trace --attr targets.status-go.mobile.android  default.nix
this derivation will be built:
  /nix/store/0nrrm7g97z4kyb1n3q1aad164gx8h5li-status-go-0.94.10-598b837-android.drv
building '/nix/store/0nrrm7g97z4kyb1n3q1aad164gx8h5li-status-go-0.94.10-598b837-android.drv'...
unpacking sources
unpacking source archive /nix/store/b1cxqjh9dsjn4w5llg3ig7kn3662icni-status-go-598b837-source
source root is status-go-598b837-source
patching sources
configuring
building

Building for targets: android/arm,android/arm64,android/386

OPENSEA_API_KEY: 
```
So why @churik identified it as working is beyond me.

Reply:
added `v1.3.5`